lib/ash/actions/read/read.ex

```diff
@@ -1143,7 +1142,26 @@ defmodule Ash.Actions.Read do
   end
 
   @doc false
-  def handle_attribute_multitenancy(query) do
+  def handle_multitenancy(query) do
+    action_multitenancy = get_action(query.resource, query.action).multitenancy
+
+    case action_multitenancy do
+      :enforce ->
+        query = handle_attribute_multitenancy(query)
+
+        with :ok <- validate_multitenancy(query) do
+          {:ok, query}
+        end
+
+      :allow_global ->
+        {:ok, handle_attribute_multitenancy(query)}
+
+      :bypass ->
+        {:ok, %{query | tenant: nil, to_tenant: nil}}
+    end
+  end
+
+  defp handle_attribute_multitenancy(query) do
```

Comment:
This was public but with `doc false`, is it ok to make it private given they're not called anymore? Same for `validate_multitenancy`

Reply:
yep!